examples/synthetics_mesh_example_test.go

```diff
@@ -152,3 +156,131 @@ func (m metricsMatrix) getMetric(fromAgent string, toAgent string) (*synthetics.
 
 	return metric, true
 }
+
+func TestGetMeshTestResultsGRPCExample(t *testing.T) {
+	assert.NoError(t, runGetMeshTestResultsGRPC())
+}
+
+func runGetMeshTestResultsGRPC() error {
+	flag.Parse()
+
+	mesh, err := getMeshTestResultsGRPC(*testID)
+	if mesh == nil {
+		fmt.Println("Empty mesh test result received")
+	} else {
+		metricsMatrix := newMetricsMatrixGRPC(mesh)
+		printMetricsMatrixGRPC(metricsMatrix)
+	}
+
+	return err
+}
+
+func getMeshTestResultsGRPC(testID string) ([]*syntheticspb.MeshResponse, error) {
+	client, err := NewClient()
+	if err != nil {
+		return nil, err
+	}
+
+	healthPayload := &syntheticspb.GetHealthForTestsRequest{
+		Ids:       []string{testID},
+		StartTime: timestamppb.New(time.Now().Add(-time.Minute * 5)),
+		EndTime:   timestamppb.Now(),
+		Augment:   true, // if not set, returned Mesh pointer will be empty
```

Comment:
Despite that, all of the data you may want to display will be present (just not it the `mesh` section) :-).